Gemfile

```diff
@@ -67,5 +67,5 @@ gem "lograge"
 gem "ougai", "~> 1.7"
 gem "rake"
 gem "redis-actionpack"
-gem "sidekiq", "< 6"
+gem "sidekiq"
```

Comment:
probably supposed to be in the next commit. but that's fine.

Reply:
Yup, it was supposed to be.